### DISCUSSION TITLE
Golang IdToken Session support only nested claims

### DISCUSSION THREAD

**Discussion by kotyara85** [ory/hydra#3096] [Q&A] [answered]
Hello,
Currently Session field in the Consent accept is map[string]map[string]interface{}
```
Session: &hydra.ConsentRequestSession{
	IdToken: map[string]map[string]interface{}{
		"test": map[string]interface{}{
			"hello": "123",
		},
	},
}
```

which allows only to setup nested claims
```
{
    "test": {
        "hello": "123"
    }
}
```

Is there a way to create claims on the root level? TS examples seem to allow to do that
Thanks

**Comment by vinckr** [comment, marked as answer]
Hey kotyara85,
is this question regarding the [Ory Hydra Go SDK](https://github.com/ory/sdk/tree/master/clients/hydra/go)?

**Reply by kotyara85** [reply]
Yes, there's a bug on this, PR already opened. thanks
